lib/msf/core/post/file.rb

```diff
@@ -704,6 +704,61 @@ def _unix_max_line_length
     line_max
   end
 
+  def search( root=nil, glob='*.*', recurse=true, timeout=-1 )
```

Comment:
Could you please add the YARD documentation for this? I've found this [cheatsheet](https://gist.github.com/chetan/1827484) pretty helpful in the past while I've been trying to remember the syntax.

Reply:
Also, `timeout` probably shouldn't be a parameter since it's only being used by the Meterpreter version. You should remove it here and just pass `-1` in the Meterpreter version on L711. That will prevent callers who are using shell session from passing in a timeout thinking it's going to do something.

Reply:
cool, will do that, thanks for the cheatsheet!

Reply:
Resolving this as I've added the yard documentation for `stat` and `search` and also removed the timeout arg. :+1: